apps/web/src/hooks/useBestAMMTrade.ts

```diff
@@ -353,3 +367,185 @@ export const useBestAMMTradeFromQuoterWorker2 = bestTradeHookFactory({
   // Since quotes are fetched on chain, which relies on network IO, not calculated offchain, we don't need to further optimize
   quoterOptimization: false,
 })
+
+export enum SwapRouterNativeAssets {
```

Comment:
All the following conversion related logic can be put inside the universal router sdk. Eventually we will get rid of them and use our own type